src/extensions/workspace/workspace.provider.ts

```diff
@@ -22,15 +22,11 @@ export type WorkspaceConfig = {
 export default async function provideWorkspace(config: WorkspaceConfig, [scope, component, capsule]: WorkspaceDeps) {
   // This is wrapped since there are cases when there is no workspace, or something in the workspace is invalid
   // Those will be handled later
-  try {
-    const consumer = await loadConsumerIfExist();
-    if (consumer) {
-      const workspace = new Workspace(consumer, scope, component, capsule);
-      return workspace;
-    }
-
-    return undefined;
-  } catch {
-    return undefined;
+  const consumer = await loadConsumerIfExist();
```

Comment:
@davidfirst you removed the try-catch, it might be break when the consumer exists but with invalid bit.json / bitmap or something like this.

Reply:
@GiladShoham , I reverted Doron's code from Friday to what Ran did before. If I change it to loadConsumer, I can't run `bit init`. I think some tests about init --reset are broken and it might be related to this. I'll check.

Reply:
Fixed to be the same as harmony/main.